content/W2016/W16-media-stephanie.md

```diff
@@ -0,0 +1,16 @@
+Title: Stephanie Morillo Talk
+Date: 2016-03-22
+Category: Media
+Tags: talks
+Slug: stephanie-morillo-video-2016
+Author: Fatema Boxwala
+Summary: The amazing Stephanie Morillo gives a talk entitled Circuit Breaker
+
+You can find more information about the event
+[here]({filename}/W2016/W16-event-stephanie-talk.md).
+
+<video width="320" height="240" controls>
+  <source src="http://mirror.csclub.uwaterloo.ca/wics/stephanie_morillo.mp4"
+          type="video/mp4">
+Your browser does not support the video tag.
+</video>
```

Comment:
The deploy looks good but the page is pretty sparse. Can we add the video download link and size to the bottom of the page? Like on the Sarah talk :)